ignite/services/network/network_test.go

```diff
@@ -64,3 +65,9 @@ func TestParseID(t *testing.T) {
 		})
 	}
 }
+
+func SampleSharePercent(t *testing.T, denom string, nominator, denominator uint64) SharePercent {
+	sp, err := NewSharePercent(denom, nominator, denominator)
+	assert.NoError(t, err)
```

Comment:
```suggestion
	require.NoError(t, err)
```

So tests can fail immediately?

Reply:
With assert it also fails immediately

Reply:
Hm, as far as I remember it waits all other assertions in the test instead of failing immediately.

Reply:
Yeah you are right. It was failing for me because different package was imported by mistake 
https://github.com/ignite-hq/cli/pull/2527/files#diff-a3a5f8302b7bac7ede63953caafc705410040be1de3dbf3f6baa38b186bf209aR7